{"name": "test_mul_parameters[mul_parameters0]", "status": "failed", "statusDetails": {"message": "AssertionError: This is correct\nassert (6 * 6) == 32", "trace": "mul_parameters = (6, 6, 32)\n\n    def test_mul_parameters(mul_parameters):\n        a, b, expected = mul_parameters\n>       assert a * b == expected, \"This is correct\"\nE       AssertionError: This is correct\nE       assert (6 * 6) == 32\n\ntest_scripts\\test_pipeline.py:55: AssertionError"}, "parameters": [{"name": "mul_parameters", "value": "(6, 6, 32)"}], "start": 1746510691752, "stop": 1746510691752, "uuid": "6d1ce7af-4a55-46b8-a95f-f0f8bca44844", "historyId": "272bcf2bbe9fd79257316823f501d522", "testCaseId": "ff6efd59efa021bf2ed011068d4d2c43", "fullName": "test_scripts.test_pipeline#test_mul_parameters", "labels": [{"name": "parentSuite", "value": "test_scripts"}, {"name": "suite", "value": "test_pipeline"}, {"name": "host", "value": "LAPTOP-TR53LJF3"}, {"name": "thread", "value": "45880-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_scripts.test_pipeline"}]}